auth-web/src/components/pay/eft/ShortNameAccountLink.vue

```diff
@@ -57,50 +58,79 @@
         <template #item-slot-amountOwing="{ item }">
           <span>{{ formatCurrency(item.amountOwing) }}</span>
         </template>
-        <template #item-slot-actions="{ index }">
+        <template #expanded-item="{ item }">
+          <tr class="expanded-item-row">
+            <td
+              :colspan="headers.length"
+              class="pb-5 pl-0"
+            >
+              <span class="expanded-item scheduled-item">
```

Comment:
After discussions with Ethan, additional states for alerts were removed due to changes in how they will be displayed in the refund flow. They will be implemented as part of refund flow.